theia-rust-docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Theia with Rust support
+
+## Building the image
+
+<pre>
+cd [this repo]/theia-rust-docker
+RUST_CHANNEL=nightly make image
```

Comment:
The nightly channel does not seem to build right now. Add an example here with stable channel or without mentioning channel so that the known-to-work default in Dockerfile is used?

Reply:
As well, since make target "image" does not use "--no-cache", add an example with "make clean-image" ?

Reply:
Updated readme